server/app/lib/normalizers.js

```diff
@@ -0,0 +1,60 @@
+import neo4j from 'neo4j-driver';
+import PhoneNumber from 'awesome-phonenumber';
+import { geoCode } from './utils';
+import { ValidationError } from './errors';
+
+const WGS_84_2D = 4326;
+
+const ALLOWED_ATTRS = ['first_name', 'last_name', 'date_of_birth', 'email', 'status'];
+
+export function normalizeAddress(address) {
+  return {
+    ...address,
+    state: address.state.toUpperCase(),
+    zip: address.zip.toString().replaceAll(' ', ''),
```

Comment:
I believe replaceAll() isn't compatible with our version of node.js. I think this will have to be a replace()